<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statisticswindow</class>
 <widget class="QMainWindow" name="statisticswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ddeeff;
font-family: &quot;Arial&quot;, sans-serif;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 249, 244);
padding: 20px;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 14px;
	background-color: #ddeeff;
    font-weight: 500;
    color: #2c3e50;
    min-width: 180px;
}</string>
    </property>
    <property name="text">
     <string>Введите дату начала рассчета:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>190</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #7abdff;
    color: white;
    border: 1px solid #5ca9e6;
    border-radius: 6px;
    padding: 12px 24px;
    font-size: 14px;
    font-weight: 500;
    min-width: 140px;
}
QPushButton:hover {
    background-color: #2980b9;
    border: 1px solid #1e6da3;
}
QPushButton:pressed {
    background-color: #1a6ea7;
    padding-top: 13px;
    padding-bottom: 11px;
}</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>480</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #7abdff;
    color: white;
    border: 1px solid #5ca9e6;
    border-radius: 6px;
    padding: 12px 24px;
    font-size: 14px;
    font-weight: 500;
    min-width: 140px;
}
QPushButton:hover {
    background-color: #2980b9;
    border: 1px solid #1e6da3;
}
QPushButton:pressed {
    background-color: #1a6ea7;
    padding-top: 13px;
    padding-bottom: 11px;
}</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #dfe6e9;
border-radius: 6px;
padding: 10px 15px;
font-size: 14px;
min-width: 120px;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-size: 14px;
	background-color: #ddeeff;
    font-weight: 500;
    color: #2c3e50;
    min-width: 180px;
}</string>
       </property>
       <property name="text">
        <string>Среднее систол. давление</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">border: 1px solid #dfe6e9;
border-radius: 6px;
padding: 10px 15px;
font-size: 14px;
min-width: 120px;
background-color: white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-size: 14px;
	background-color: #ddeeff;
    font-weight: 500;
    color: #2c3e50;
    min-width: 180px;
}</string>
       </property>
       <property name="text">
        <string>Среднее диастол. давление</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="styleSheet">
        <string notr="true">border: 1px solid #dfe6e9;
border-radius: 6px;
padding: 10px 15px;
font-size: 14px;
min-width: 120px;
background-color: white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>222</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-size: 14px;
	background-color: #ddeeff;
    font-weight: 500;
    color: #2c3e50;
    min-width: 180px;
}</string>
       </property>
       <property name="text">
        <string>Средний пульс</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_3">
       <property name="styleSheet">
        <string notr="true">border: 1px solid #dfe6e9;
border-radius: 6px;
padding: 10px 15px;
font-size: 14px;
min-width: 120px;
background-color: white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
